bedrock/src/main/resources/addon/resource_pack/texts/zh_CN.lang

```diff
@@ -0,0 +1,3 @@
+tile.lucky:lucky_block.name=幸运方块
```

Comment:
this is .lang please reformat this to  .json and this it label names are incorrect it is block.luckyblock not 
tile.luckyblock and also add the translation for the item as well 😄